src/gosling-track/gosling-track.ts

```diff
@@ -1233,6 +1236,39 @@ function GoslingTrack(HGC: import('@higlass/types').HGC, ...args: any[]): any {
         }
 
         getMouseOverHtml(mouseX: number, mouseY: number) {
+            // `trackClick` API
+            const [x, y] = this.position;
+            const [width, height] = this.dimensions;
+            if (this.options.spec.layout === 'circular') {
+                const cx = x + width / 2.0;
+                const cy = y + height / 2.0;
+                const innerRadius = this.options.spec.innerRadius!;
+                const outerRadius = this.options.spec.outerRadius!;
+                const startAngle = this.options.spec.startAngle!;
+                const endAngle = this.options.spec.endAngle!;
+                // Call the API function only when the mouse is positioned directly on the track display area
+                if (
+                    isPointInsideDonutSlice(
+                        [mouseX, mouseY],
+                        [width / 2.0, height / 2.0],
+                        [innerRadius, outerRadius],
+                        [startAngle, endAngle]
+                    )
+                ) {
+                    publish('trackMouseOver', {
+                        id: this.viewUid,
+                        spec: JSON.parse(JSON.stringify(this.options.spec)),
```

Comment:
I believe you could just use the newish [`structuredClone`](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) method here:

```suggestion
                        spec: structuredClone(this.options.spec),
```